modules/gdscript/gdscript_vm.cpp

```diff
@@ -397,7 +397,13 @@ void (*type_init_function_table[])(Variant *) = {
 #define OPCODES_END
 #define OPCODES_OUT
 #define DISPATCH_OPCODE continue
+#ifdef MSVC
```

Comment:
Does this work? I think usually we use `_MSC_VER`.

I see only one occurrence of `#ifdef MSVC` in the codebase, which might be a bug.

Reply:
It is defined:
https://github.com/godotengine/godot/blob/df0a822323a79e1a645f0c6a17d51c7602f23166/platform/windows/detect.py#L382-L388

Reply:
It seems a bit unnecessary to define it here, to use it only in a single location, when we can rely on `_MSC_VER` like 500 other locations in the codebase :)

So I'd suggest removing that define.

Reply:
Pushed with use of `_MSC_VER`.